### PR TITLE
Updated docs

### PR DESCRIPTION
**Pull request by Nyholm**
I ran the commands

```
curl -O http://get.sensiolabs.org/sami.phar
php sami.phar update sami.php
```

### CODE REVIEW

**Comment by stof** [comment]
why is there no doc for the older release anymore ?

Btw, the sami.php config in the project should be updated. `->addFromTags('v0.6.0')` should be changed, to allow documenting the latest stable release

**Comment by stof** [comment]
Please also update the URLs in the index.html at the root of gh-pages, as many of them point to the wrong repo (not pointing to the org)

**Comment by Nyholm** [comment]
Using `->addFromTags('v0.6.0')` resulted in invalid docs for master.. I will give this another try then.

File: index.html [hunk 1]
```diff
@@ -89,7 +89,7 @@
 
 <ul>
 <li>
-<a href="http://github.com/henrikbjorn/HBStampieBundle">HBStampieBundle</a> it is also <a href="http://packagist.org/packages/henrikbjorn/stampie-bundle">on packagist</a>
+<a href="http://github.com/Stampie/HBStampieBundle">HBStampieBundle</a> it is also <a href="http://packagist.org/packages/henrikbjorn/stampie-bundle">on packagist</a>
```
Comment (stof) [inline index.html]: Can you use HTTPS links too ?

File: index.html [hunk 2]
```diff
@@ -18,19 +18,19 @@
...
+          <li><a href="https://github.com/Stampie/Stampie/zipball/master">Download <strong>ZIP File</strong></a></li>
+          <li><a href="https://github.com/Stampie/Stampie/tarball/master">Download <strong>TAR Ball</strong></a></li>
+          <li><a href="https://github.com/Stampie/Stampie">View On <strong>GitHub</strong></a></li>
         </ul>
       </header>
       <section>
         <h1>Stampie</h1>
 
-<p><a href="http://travis-ci.org/henrikbjorn/Stampie"><img src="https://secure.travis-ci.org/henrikbjorn/Stampie.png" alt="Build Status"></a></p>
+<p><a href="http://travis-ci.org/Stampie/Stampie"><img src="https://secure.travis-ci.org/Stampie/Stampie.png" alt="Build Status"></a></p>
```
Comment (stof) [inline index.html]: should use HTTPS, and the SVg badge

**Pull request merged**